.github/workflows/buildsCI.yaml

```diff
@@ -47,60 +45,33 @@ jobs:
        mkdir ${{env.WIN_BUILD_DIR}}
     
     - name: Configure CMake
-    # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
-    # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
       shell: bash
       run: |        
-        LRS_SRC_DIR=$(pwd)   
+        LRS_SRC_DIR=$(pwd)
         cd ${{env.WIN_BUILD_DIR}}
         pwd
         ls        
-        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DPYTHON_EXECUTABLE=${{env.PYTHON_PATH}} -DBUILD_PYTHON_BINDINGS=true -DPYBIND11_PYTHON_VERSION=3.8 -DBUILD_UNIT_TESTS=false -DBUILD_LEGACY_LIVE_TEST=true -DBUILD_EXAMPLES=false -DBUILD_TOOLS=true -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true -DBUILD_WITH_DDS=true
+        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DBUILD_WITH_TM2=false -DCHECK_FOR_UPDATES=true
 
     - name: Build
       # Build your program with the given configuration
       shell: bash
       run: |
         cd ${{env.WIN_BUILD_DIR}}
-        pwd
-        ls
         cmake --build . --config ${{env.LRS_RUN_CONFIG}} -- -m    
-     
-    - name: Test
-      shell: bash
-      id: test-step
-      # We set continue-on-error: true as a workaround for not skipping the upload log step on failure,
-      # The final step will check the return value from the test step and decide if to fail/pass the job
-      continue-on-error: true 
-      run: |
-       export LRS_LOG_LEVEL="DEBUG";
-       cd  ${{env.WIN_BUILD_DIR}}/${{env.LRS_RUN_CONFIG}}
-       ls
-       ./live-test.exe -d yes -i [software-device]  
-       
-    - name: Upload RS log artifact
-      uses: actions/upload-artifact@v2
-      with: 
-        name: Log file - Windows_testing_cpp_USB_Win7
-        path:  ${{env.WIN_BUILD_DIR}}/${{env.LRS_RUN_CONFIG}}/*.log
-        
-    - name: Provide correct exit status for job 
-      shell: bash
-      run: |
-        if [ ${{steps.test-step.outcome}} = "failure" ];then
-           echo "Test step failed, please open it to view the reported issue"
-           exit 1
-        else
-           exit 0
-        fi
-  
-  Windows_cpp:
+
+
+  #--------------------------------------------------------------------------------
+  Win_ST_Py_DDS:  # Windows, Static, Python, Tools, DDS
```

Comment:
What do you think, maybe we can run some non live python test here?
Just to make sure we have WIN static build running OK?

Otherwise if we break something we won't know.. libCI run with shared libs.

Reply:
I have plans, but not for this PR.

Reply:
OK, I'll be happy to hear about the plans :)